source/dinotify.d

```diff
@@ -273,11 +275,11 @@ public struct INotifyTree
     }   
 }
 
-/// Create a INotifyTree to recusivly establish watches in `path`,
+/// Create a INotifyTree to recusivly establish watches in `roots`,
 /// using `mask` to choose which events to watch on.
-public auto iNotifyTree(string path, uint mask)
+public auto iNotifyTree(uint mask, string[] roots...)
```

Comment:
Ah so it must be past parameter?
In such case sorry for the mess, please revert to your original version and I’ll merge.

Rationale: we should not break existing code (however few users this library has).

Reply:
yes ... break existing code is not so nice. i was wondering about that... how widely is dinotify used?
i am also wondering why it is, that the variadic thing needs to be last? especially if there are different types used as parameters.

Reply:
I’m not sure about the scope of user but it’s > 0 and that is enough for me to avoid breaking API without very strong reason.